<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_des_equipements</class>
 <widget class="QDialog" name="gestion_des_equipements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Equipement</string>
      </attribute>
      <widget class="QTableView" name="tableView">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>40</y>
         <width>561</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>261</width>
         <height>227</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>supprimer</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Imprimer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Historique</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string>choisir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Statistique</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>291</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Type :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="typeequi"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nomeq"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="id_eq"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Id_Equipemnet :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLineEdit" name="idfourr">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_3"/>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Id_Fournisseur :</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Equipement :</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>250</y>
         <width>561</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>10</y>
         <width>334</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Recherche :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Trier par :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_4">
            <property name="text">
             <string>Z-A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_3">
            <property name="text">
             <string>A-Z</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Fournisseur</string>
      </attribute>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(85, 170, 255);</string>
       </property>
       <property name="text">
        <string>Fournisseur :</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>321</width>
         <height>172</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>choisir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2"/>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="modifier_four">
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="ajouter_four">
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Historique</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Imprimer</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="supprimer_four">
          <property name="text">
           <string>Supprimer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView_2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>551</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>365</x>
         <y>270</y>
         <width>541</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>331</width>
         <height>203</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="1">
         <widget class="QLineEdit" name="emioak"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>E_mail :</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Nom fournisseur :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Adresse :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="id_four"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="tel"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nomfour"/>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="adres"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Id_Fournisseur :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Tel :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Commentaire :</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QTextEdit" name="dddd"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>10</y>
         <width>334</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Trier par :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>A-Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Z-A</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Recherche :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
